src/Game/GameObjects/Views/MobileView.cs

```diff
@@ -96,10 +98,18 @@ private void DrawCharacter(UltimaBatcher2D batcher, int posX, int posY)
 
             bool hasShadow = !IsDead && !IsHidden && Engine.Profile.Current.ShadowsEnabled;
 
-
             if (Engine.AuraManager.IsEnabled)
-                Engine.AuraManager.Draw(batcher, drawX + 22, drawY + 22, Notoriety.GetHue(NotorietyFlag));
-
+            { 
+                if (World.Party.Contains(this) && Engine.Profile.Current.PartyAura)
```

Comment:
Just an optmiziation.
It's much better to evalutate `Engine.Profile.Current.PartyAura` before, cause the World.Party.Contains do a cycle that in the worst case is length 10.